test/hotspot/jtreg/ProblemList-svc-vthread.txt

```diff
@@ -24,8 +24,6 @@
 ####
 # Bugs
 
-vmTestbase/nsk/jdb/kill/kill001/kill001.java                00000000 generic-all
-
```

Comment:
This test was problem listed on mac-osx (the reason was this bug https://bugs.openjdk.java.net/browse/JDK-8282383) before Leonid problem listed it generally. So we want to restore this? Also, this test is failing with timeout when I run it locally on linux-x64-debug.